### PR TITLE
[sc-5895] add low-resources preset

### PR DESCRIPTION
**Pull request by steb-veksler**

### CODE REVIEW

**Review by noamApps — COMMENTED** [review]
tested on ? (kind/minikube/k3d)

**Comment by steb-veksler** [comment]
> tested on ? (kind/minikube/k3d)

k3s is working, kind and minikube have ring buffer errors on flora

**Review by noamApps — CHANGES_REQUESTED** [review]
remove vmagent-backend

File: cmd/deploy.go [hunk 1]
```diff
@@ -405,7 +416,7 @@ func getChartValues(chartValues map[string]interface{}, clusterName string, depl
 		"global":               map[string]interface{}{"groundcover_token": apiKey.ApiKey},
 	}
 
-	if err = mergo.Merge(&chartValues, defaultChartValues); err != nil {
+	if err = mergo.Merge(&chartValues, defaultChartValues, mergo.WithSliceDeepCopy); err != nil {
```
Comment (noamApps) [inline cmd/deploy.go]: remove from pr

File: cmd/deploy.go [hunk 2]
```diff
@@ -420,6 +431,28 @@ func getChartValues(chartValues map[string]interface{}, clusterName string, depl
...
+	if viper.GetBool(LOW_RESOURCES_FLAG) {
+		resourcesTunerPresetPaths = []string{
+			helm.AGENT_LOW_RESOURCES_PATH,
+			helm.BACKEND_LOW_RESOURCES_PATH,
+		}
+	}
+
+	if slices.Contains(resourcesTunerPresetPaths, helm.AGENT_LOW_RESOURCES_PATH) {
+		clusterType := "low resources"
+		messageFormat := "We get it, you like things light 🪁\n   But since you’re deploying on a %s we’ll have to limit some of our features to make sure it’s smooth sailing.\n   For the full groundcover experience, try deploying on a different cluster\n"
```
Comment (noamApps) [inline cmd/deploy.go:444]: move to const

**Review by noamApps — CHANGES_REQUESTED** [review]

**Review by noamApps — APPROVED** [review]

**Pull request merged**